synapse/tests/test_model_person.py

```diff
@@ -313,3 +313,18 @@ async def test_ps_vitals(self):
 
             self.len(1, await core.nodes('ps:person :vitals -> ps:vitals'))
             self.len(1, await core.nodes('ps:contact :vitals -> ps:vitals'))
+
+    async def test_ps_skillz(self):
+        async with self.getTestCore() as core:
+            nodes = await core.nodes('''
+                [ ps:proficiency=*
+                    :contact = {[ ps:contact=* :name=visi ]}
+                    :skill = {[ ps:skill=* :type=hungry :name="Wanting Pizza" ]}
+                ]
+            ''')
+            self.len(1, nodes)
+            self.nn(nodes[0].get('skill'))
+            self.nn(nodes[0].get('contact'))
+            self.len(1, await core.nodes('ps:proficiency -> ps:contact +:name=visi'))
+            self.len(1, await core.nodes('ps:proficiency -> ps:skill +:name="wanting pizza"'))
+            self.len(1, await core.nodes('ps:proficiency -> ps:skill ps:skill:type:taxonomy'))
```

Comment:
is that supposed to be a filter rather than additional lift?

Reply:
supposed to be a pivot. updated. with any luck this one will pass lol.